GroceryList/app/src/main/java/com/mtruehle/grocerylist/MainActivity.java

```diff
@@ -0,0 +1,152 @@
+package com.mtruehle.grocerylist;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.EditText;
+import android.widget.LinearLayout;
+import android.widget.ListView;
+import android.widget.Toast;
+
+import java.util.ArrayList;
+
+public class MainActivity extends AppCompatActivity {
+    private ArrayList<String> itemsToBuy;
+    private ArrayAdapter<String> itemsAdapter; // referred to by class methods.
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        // starts up the main activity, defines necessary parameters like itemsToBuy, starts up the ListView, and adds Lucky Charms to make testing stuff out slightly quicker.
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        itemsToBuy = new ArrayList<String>();
+        itemsToBuy.add("Lucky Charms"); // they're magically delicious.
+        itemsAdapter = new ArrayAdapter<String>(this, android.R.layout.simple_list_item_1, itemsToBuy);
+        ListView itemListView = (ListView) findViewById(R.id.groceryList);
+        itemListView.setAdapter(itemsAdapter);
+        itemListView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
```

Comment:
This will become cleaner if you create an implementation of OnItemClickListener in a separate file like is done here
https://github.com/OlinMobileProto/Lab1/blob/master/app/src/main/java/com/example/cwallace/grocerylist/clickListeners/DeleteEntryListener.java